src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java

```diff
@@ -173,6 +178,16 @@ public String toString() {
             .filter(si -> si.getOcrData() != null);
     }
 
+    private String findSubtypeOfScannableItemWithFormType(Envelope envelope) {
```

Comment:
should this be done after all OCR related validations has passed ?

Reply:
@nitinprabhu yes, this is done after the ocr validations have passed and the envelope is processed
https://github.com/hmcts/bulk-scan-processor/blob/2682da5a7e5af553d9acb9784fb428de25401ca6/src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/OrchestratorNotificationTask.java#L59